src/Calculator.ts

```diff
@@ -116,20 +118,109 @@ export function calculate(
+    report.type = R4.MeasureReportTypeKind._individual; // are there cases where this should be _subjectList or _summary or _dataCollection
+
+    // measure url from measure bundle
+    const measureEntry = measureBundle.entry?.find(e => e.resource?.resourceType === 'Measure') as R4.IBundle_Entry;
+    const measure = measureEntry.resource as R4.IMeasure;
+    report.measure = measure.url || 'UnknownMeasure'; // or some other default?
+
+    // create group population counts from result's detailedResults (yes/no->1/0)
+    report.group = [];
+    result?.detailedResults?.forEach(function (dr) {
+      // TODO: check the measure definition for stratification to determine whether to add group.stratiier
+      // if yes, add stratifier with population copied into. Set counts to 0 if the result for the stratifier is false
+      const group: R4.IMeasureReport_Group = <any>{};
+      const detail: DetailedPopulationGroupResult = dr;
+      group.id = detail.groupId;
+      group.population = [];
+      let numeratorCount = 0.0;
+      let denominatorCount = 0.0;
+      // TODO: handle EXM111 (doesn't identify itself as a episode of care measure). if it's an episode of care, you need to iterate over
+      // stratifications : may need to clone results for one population group and adjust (in this case, just a straight clone)
+      detail?.populationResults?.forEach(function (pr) {
+        const pop: IMeasureReport_Population = <any>{};
+
+        pop.code = {
+          coding: [
+            {
+              system: 'http://terminology.hl7.org/CodeSystem/measure-population',
+              code: pr.populationType,
+              display: populationDisplay(pr.populationType)
+            }
+          ]
+        };
+        pop.count = pr.result ? 1 : 0;
+        if (pr.populationType == PopulationType.NUMER) numeratorCount += pop.count;
+        if (pr.populationType == PopulationType.DENOM) denominatorCount += pop.count;
+        group.population?.push?.(pop);
+      });
+      // TODO: handle ratio or continuous variable or cohort
+      // currently assumes patient-based proportion measure http://hl7.org/fhir/us/cqfmeasures/measure-conformance.html#proportion-measures
+      group.measureScore = {
+        value: (numeratorCount / denominatorCount) * 1.0
+      };
+      report.group?.push(group);
+    });
+
+    // TODO: create contained evalatuated resource (contains patient... and population and entity information?)
+    // may eventually need the other bits (all pieces of information that were used in calculation), but we might be able to ignore for now
+    const evalId = uuidv4();
+    const contained: R4.IBundle = {
+      resourceType: 'Bundle',
+      id: evalId,
+      type: R4.BundleTypeKind._collection,
+      entry: []
+    };
+    const patient = patientBundles[0].entry?.filter(function (e) {
+      return e.resource?.resourceType == 'Patient';
+    })[0].resource;
```

Comment:
```suggestion
    const patient = patientBundles[0].entry?.find((e) => {
      return e.resource?.resourceType === 'Patient';
    }).resource;
```

If you use `find` here instead, you shouldn't need to index using `[0]` since `find` will always grab the first occurrence of something. The condition in the function can stay the same.

If we think we might need other resources, you can ignore this suggestion